cupy/core/core.pyx

```diff
@@ -2812,9 +2812,15 @@ cpdef ndarray tensordot_core(
             out = _ndarray_init(ret_shape, dtype)
 
     if m == 1 and n == 1:
-        # TODO(leofang): switch back to _tensordot_core_mul_sum when
-        # ReductionKernel supports CUB block reduction
-        out = (a.ravel() * b.ravel()).sum(out=_manipulation._reshape(out, ()))
+        if len(_accelerator._routine_accelerators) > 0:
+            # fast path using CUB or cuTENSOR; if not applicable, it'd fall
+            # back to CuPy's reduction kernel
```

Comment:
The implementation of this PR fallback to CuPy's default `(a.ravel() * b.ravel()).sum(...)`, not reduction kernel.

Reply:
Oh, by this I meant `sum` is backed by CuPy's reduction kernel implemented in `cupy/core/_reduction.pyx`, not the `ReductionKernel` object. Would you like me to make it clearer?

Reply:
How about `CuPy's default sum kernel`?

Reply:
Sounds good.